rplugin/python3/semshi/handler.py

```diff
@@ -14,6 +15,15 @@
 ERROR_HL_ID = 313000
 
 
+@dataclasses.dataclass(frozen=True)
+class ViewPort:
+    """Start-end line of a window for (this) buffer
+
+    """
+    start:int
+    end:int
```

Comment:
This would require python 3.7. Although python 3.6 is EOD, can we use namedtuple or some plain class instead?